### ISSUE TITLE
Error: 404 Not Found - GET https://registry.npmjs.org/@types/eslint/-/eslint-8.4.4.tgzBug: (fill in)

### ISSUE THREAD

**Issue by fredDesign** [eslint/eslint#16077]
### Environment

Node version: 
npm version: 
Local ESLint version: 8.4.4
Global ESLint version: 8.4.4
Operating System: 

### What parser are you using?

Other

### What did you do?

<details>
<summary>Configuration</summary>

```
<!-- Paste your configuration here -->
```
</details>

```js
<!-- Paste your code here -->
```

### What did you expect to happen?

install laravel mix package

### What actually happened?

404 on eslint 8.4.4

### Participation

- [ ] I am willing to submit a pull request for this issue.

**Comment by smerkousdavid** [comment]
This isn't an issue with eslint but rather @types/eslint. Please checkout https://github.com/DefinitelyTyped/DefinitelyTyped/issues/61032

**Comment by lucasff** [comment]
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/@types/eslint/-/eslint-8.4.4.tgz
npm ERR! 404 
npm ERR! 404  '@types/eslint@*' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404 It was specified as a dependency of '@types/eslint-scope'
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

**Comment by mdjermanovic** [comment]
Duplicate of https://github.com/eslint/eslint/issues/16074.

As mentioned, this isn't an issue with ESLint.
